### PR TITLE
KEYBOARD ADDITION: IV Works Umbra

### PR DESCRIPTION
**Pull request by josephawilliamsiv**
## Description

Files to add new keyboard "Umbra" to QMK.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by waffle87 — COMMENTED** [review]
It would be preferable to have this under a single keyboard, with different `hotswap` and `solder` subfolders that contain configuration specific to each version, and common configuration shared between both PCBs at the top keyboard level. Like
```shell
umbra
├── config.h
├── hotswap
│   ├── keyboard.json
│   └── keymaps
│       └── default
├── info.json
├── readme.md
└── solder
    ├── keyboard.json
    └── keymaps
        └── default
```

**Comment by josephawilliamsiv** [comment]
Great feedback! Made the changes to file structure.

**Comment by josephawilliamsiv** [comment]
Not sure why the checks would have failed. Any advice?

File: keyboards/umbra/rules.mk
Comment (zvecr) [inline keyboards/umbra/rules.mk:1]: > Not sure why the checks would have failed. Any advice?

Its this files existence. Its existence creates a build target, that cannot be built.

The file should be duplicated to each revision, and this one removed.

**Pull request closed without merging**